.github/.kubescape/exceptions.json

```
@@ -14,5 +14,167 @@
         }
       }
     ]
+  },
+  {
+    "name": "application-credentials-in-configuration-files",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "Deployment"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0012"
+        }
+    ]
+  },
+  {
+    "name": "images-allowed-registry",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "Deployment"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0078"
+        }
+    ]
+  },
+  {
+    "name": "auto-map-service-account-nats",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "ServiceAccount",
+          "name": "keptn-nats"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0034"
+        }
+    ]
+  },
+  {
+    "name": "auto-map-service-account-mongo",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "Deployment",
+          "name": "keptn-mongo"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0034"
+        }
+    ]
+  },
+  {
+    "name": "ingress-egress-blocked-deployment",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "Deployment"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0030"
+        }
+    ]
+  },
+  {
+    "name": "ingress-egress-blocked-statefulset",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "StatefulSet"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0030"
+        }
+    ]
+  },
```

you can probably combine those 2 rules with 1 resource each into 1 rule with 2 resources

```suggestion
  {
    "name": "ingress-egress-blocked",
    "policyType": "postureExceptionPolicy",
    "actions": [
      "alertOnly"
    ],
    "resources": [
      {
        "designatorType": "Attributes",
        "attributes": {
          "kind": "Deployment"
        }
      },
      {
        "designatorType": "Attributes",
        "attributes": {
          "kind": "StatefulSet"
        }
      }
    ],
    "posturePolicies": [
        {
            "controlID": "C-0030"
        }
    ]
  },
```